Makefile.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "check-rs-format",
     "check-ts-format",
     "check-ts",
+    "check-cargo-toml-format",
```

Comment:
Why the `cargo`?  Perhaps `check-toml-format`

Reply:
This tool is meant to format `Cargo.toml` files, not any TOML file. It actually fails on random TOML files:
```
▶ cargo tomlfmt -d  -p Makefile.toml 
 WARN 2019-07-11T02:09:39Z: cargo_tomlfmt: dryrun founds problems in Cargo.toml
 WARN 2019-07-11T02:09:39Z: cargo_tomlfmt: exit with -1
```